families/seth/src/sawtooth_seth/cli/permissions_set.go

```diff
@@ -32,6 +32,7 @@ type PermissionsSet struct {
 	Address     string `short:"a" long:"address" required:"true" description:"Address of account whose permissions are being changed; 'global' may be used to refer to the zero address"`
 	Permissions string `short:"p" long:"permissions" required:"true" description:"New permissions for the account"`
 	Nonce       uint64 `short:"n" long:"nonce" description:"Current nonce of the moderator account" default:"1"`
+	Wait   		int    `short:"w" long:"wait" description:"Number of seconds Seth client will wait for transaction to be committed" default:"0"`
```

Comment:
Looks like an extra tab here

Reply:
Was `gofmt` run on the code?

Reply:
yes, it seemed to pass despite the extra tabs

Reply:
Misunderstood how gofmt worked. I will run it and integrate the changes in the fix